lib/worker/subprocess.js

```diff
@@ -111,7 +111,7 @@ ipc.options.then(options => {
 			const required = require(mod);
 
 			try {
-				if (required[Symbol.for('esm\u200D:package')]) {
+				if (required[Symbol.for('esm:package')]) {
```

Comment:
I think we need to check for both symbols, to maintain compatibility with projects using older `esm` versions. Can check for the new symbol first of course.

Reply:
Ah gross. I didn't intend to make more work for y'all. Okay. I'll mod it to check box and y'all can drop the legacy one when you all do a major bump.

Reply:
Updated ✨